### PR TITLE
content modified

### PR DESCRIPTION
**Pull request by soujo**
## Changes
- Improved content of `contributing.md` 
- Added content to `PULL_REQUEST_TEMPLATE.md`

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -14,6 +14,32 @@ Contributing is also a great way to learn more about social coding on Github, ne
     - Good commit message: `updated contributors list in readme`
 - You **should not**, in any case, use resources or code snippets from sources that do not allow their public use.
 
+
+## Pull Requests Review Criteria
+
+- Please fill the **[PR Template]()** properly while making a Pull Request.
+
+- You must add your script into the respective **folders**.
```
Comment (singhayushh) [inline contributing.md]: Good changes overall, but why have you mentioned "script".

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,7 +1,48 @@
-<!-- Add Issue Number, if linked -->
-Linked Issue: 
+## Related Issue 
 
-<!-- Add change log below -->
-## Changes
--
--
+- Info about the related issue
```
Comment (singhayushh) [inline .github/PULL_REQUEST_TEMPLATE.md]: Could you remove this section entirely - Info about the related issue. Just linking the issue would do for us.

**Review by singhayushh — CHANGES_REQUESTED** [review]

**Comment by singhayushh** [comment]
Thank you for taking out time and contributing to this repo!

**Comment by soujo** [comment]
Required changes have been added, kindly check

**Review by Codehackerone — DISMISSED** [review]
@soujo Can you add Issue templates also? 
Not just specific for KSoC.

**Review by Codehackerone — APPROVED** [review]

**Review by singhayushh — APPROVED** [review]

**Pull request merged**